package.json

```diff
@@ -66,7 +66,7 @@
     "ts-loader": "^6.2.1"
   },
   "engines": {
-    "yarn": "^1.21.1"
+    "yarn": "1.21.1"
```

Comment:
This change should not happen. Please leave it as `^1.21.1`. This plugin would only and only change this version if this plugin needs something from Yarn that the previous version did not offer, and also it has to always be a range and never a specific version.

Reply:
aah, I should have reverted this